azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -648,6 +652,9 @@ async def rewind(self,
         elif status == 410:
             ex_msg = "The rewind operation is only supported on failed orchestration instances."
             raise Exception(ex_msg)
-        else:
+        elif isinstance(response[1], str):
```

Comment:
Just to be clear, as someone who has rusty Python, this is checking that we have an element in response[1]?

Reply:
Not quite, this is checking that the element in `response[1]` is of type "string",
We already know that `response` is an array of 2 elements because of the definition of `__post_request_async` which ultimately defers to this definition: https://github.com/Azure/azure-functions-durable-python/blob/dev/azure/durable_functions/models/utils/http_utils.py#L29